pkg/authentication/handlers.go

```diff
@@ -17,15 +17,38 @@ import (
 	"github.com/canonical/identity-platform-admin-ui/internal/validation"
 )
 
+const (
+	codeParameter  = "code"
+	stateParameter = "state"
+
+	codeParamNotFoundMsg   = "OAuth2 code not found"
+	stateParamNotFoundMsg  = "OAuth2 state not found"
+	noIDTokenMsg           = "unable to retrieve ID token"
+	nonceCookieNotFoundMsg = "nonce cookie not found"
+	idTokenNonceMatchMsg   = "id token nonce does not match nonce cookie"
+	stateCookieNotFoundMsg = "state cookie not found"
+	stateParamMatchMsg     = "state parameter does not match state cookie"
+)
+
+var (
```

Comment:
Use `const` instead?

Reply:
I wish!
Unfortunately the "amazing" Go language doesn't allow to have constant errors.

Reply:
TIL